Classes/common/CDTDatastore+Conflicts.m

```diff
@@ -35,18 +35,16 @@ @implementation CDTDatastore (Conflicts)
 
 - (NSArray *)getConflictedDocumentIds
 {
-    if (![self.database open]) {
-        return nil;
-    }
-
-    return [self.database getConflictedDocumentIds];
+    // This property is not synthesized, it is a method that already ensures that the
+    // database is open (or return nil)
+    return (self.database ? [self.database getConflictedDocumentIds] : nil);
```

Comment:
Well, when a CDTDatastore is created, the database passed as a parameter is open so it shouldn't be necessary to check if it is already open all the time but the database is a public property, so I assumed (while I was modifying the code) that there are some cases where the database is closed from outside this class.